### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated filename

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/564)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 # [](https://github.com/rudderlabs/rudder-sdk-js/blob/master/README.md#rudderstack-javascript-sdk)RudderStack JavaScript SDK
 
-The [**RudderStack**](https://rudderstack.com/) JavaScript SDK leverages the `rudder-analytics.js` library to track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.
+The [**RudderStack**](https://rudderstack.com/) JavaScript SDK leverages the `analytics.js` library to track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.
```
Comment (saikumarrs) [inline README.md:23]: The first sentence here doesn't make any sense to me. We should rather just say this,

```
The RudderStack JavaScript SDK helps you to track and send user events...
```

Comment (saikumarrs) [reply README.md:23]: As there is no such thing as `analytics.js` or `rudder-analytics.js` library. It's just a source code file in this repository and contributes to the final SDK.

Comment (ameypv-rudder) [inline README.md:23]:
```suggestion
The RudderStack JavaScript SDK lets you track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.
```

Comment (ameypv-rudder) [reply README.md]: got it - thanks @saikumarrs - I've made the necessary update.

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 # [](https://github.com/rudderlabs/rudder-sdk-js/blob/master/README.md#rudderstack-javascript-sdk)RudderStack JavaScript SDK
 
-The [**RudderStack**](https://rudderstack.com/) JavaScript SDK leverages the `rudder-analytics.js` library to track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.
+The RudderStack JavaScript SDK lets you track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.
```
Comment (ameypv-rudder) [inline README.md:23]:
```suggestion
The JavaScript SDK lets you track customer event data from your websites and send it to your specified destinations via RudderStack.
```

File: README.md [hunk 3]
```diff
@@ -20,7 +20,7 @@
 
 # [](https://github.com/rudderlabs/rudder-sdk-js/blob/master/README.md#rudderstack-javascript-sdk)RudderStack JavaScript SDK
 
-The [**RudderStack**](https://rudderstack.com/) JavaScript SDK leverages the `rudder-analytics.js` library to track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.
+The JavaScript SDK lets you track customer event data from your websites and send it to your specified destinations via RudderStack.
```
Comment (ameypv-rudder) [inline README.md:23]:
```suggestion
The JavaScript SDK lets you track customer event data from your website and send it to your specified destinations via RudderStack.
```

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**